src/snippets/react-jss.js

```diff
@@ -0,0 +1,189 @@
+    left: '1rem',
+    zIndex: '2'
+  },
+  stripes: {
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    bottom: 0,
+    left: 0,
+    width: '100%',
+    height: '100%',
+    overflow: 'hidden',
+    transform: 'skewY(-8deg)',
+    transformOrigin: 0,
+    background: 'linear-gradient(-150deg, #25b57f 0%, #acf0b5 70%)'
+  },
+  stripe: {
+    height: '10vh',
+    overflow: 'hidden',
+    transform: 'skewY(-8deg)',
+    transformOrigin: 0,
+    background:
+    'linear-gradient(-150deg, rgba(255, 255, 255, 0) 40%, #ddecf7 70%)'
+  },
+  stripeBottom: {
+    position: 'absolute',
+    bottom: 0,
+    left: 0,
+    right: 0,
+    height: '35%',
+    overflow: 'hidden',
+    background: 'linear-gradient(150deg, rgba(172, 240, 181, 0) 40%, #acf0b5 70%)'
+  },
+  titleContainer: {
+    padding: '1rem'
+  },
+  title: {
+    color: 'white',
+    margin: 0,
+    padding: 0,
+    position: 'relative',
+    fontSize: '1.5rem',
+    fontWeight: '100'
+  },
+  subTitle: {
+    extend: 'title',
+    color: '#025450',
+    marginTop: '0.5rem'
+  },
+  form: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '100%',
+    padding: '2rem',
+    boxSizing: 'border-box'
+  },
+  input: {
+    display: 'block',
+    width: '100%',
+    marginBottom: '1rem',
+    padding: '1.25rem 1rem',
+    boxSizing: 'border-box',
+    borderRadius: '0.25rem',
+    border: '1px solid transparent',
+    boxShadow: '0 1px 6px rgba(0, 0, 0, 0.1)',
+    ':focus': {
+      borderColor: '#6772e5',
+      outline: 'none',
+      boxShadow: '0 1px 6px rgba(103, 114, 229, 0.5)'
+    }
+  }
+}
+
+const buttonStyles = {
+  base: {
+    display: 'block',
+    backgroundColor: props => props.disabled ? '#BBB' : '#6772e5',
+    color: 'white',
+    border: 'none',
+    width: '100%',
+    padding: '1.25rem 1rem',
+    boxSizing: 'border-box',
+    borderRadius: '0.25rem',
+    textTransform: 'uppercase',
+    boxShadow: '0 1px 6px rgba(0, 0, 0, 0.1)',
+    marginTop: '1rem'
+  }
+};
+
+const SubmitButton = injectSheet(buttonStyles)(({ classes, children }) => (
+  <button type="submit" className={classes.base}>{children}</button>
+));
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      phoneNumber: '',
+      fields: ['email', 'phoneNumber'],
+      valid: false
+    };
+  }
+
+  handleInputChange(prop) {
+    return ev => {
+      const { value } = ev.target;
+      this.setState({
+        [prop]: value,
+        valid:
+        value.length > 0 &&
+        this.state.fields.every(field => this.state[field].length > 0)
+      });
+    };
+  }
+
+  render() {
+    const { classes } = this.props;
+    const { valid } = this.state;
+
+    return (
+      <main className={classes.container}>
+        <header className={classes.header}>
+          <div className={classes.logoContainer}>
+            <Logo color="#25b57f" size={32} />
```

Comment:
I don't see where this Logo comes from

Reply:
Sorry yeah, this is some "magic" I'm injecting here -> https://github.com/DSchau/css-in-js-playground/blob/master/src/utils/eval.ts

Reply:
And I forgot the import 😱 

`import Logo from './logo';` would normally be there

Reply:
Why not simply import the "normal" way. This magic is evil)

Reply:
I don't disagree. Future enhancement!